<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBoxWindow</class>
 <widget class="QWidget" name="NewBoxWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>263</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>263</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SandboxiePlus new box</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1" colspan="4">
      <widget class="QRadioButton" name="radTemplate">
       <property name="text">
        <string>Select restriction/isolation template:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Initial sandbox configuration:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="5">
      <widget class="QLineEdit" name="txtName">
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="4">
      <widget class="QComboBox" name="cmbTemplates"/>
     </item>
     <item row="0" column="5" colspan="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="5" colspan="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1" colspan="4">
      <widget class="QRadioButton" name="radCopy">
       <property name="text">
        <string>Copy options from an existing box:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sandbox Name:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="4">
      <widget class="QComboBox" name="cmbBoxes"/>
     </item>
     <item row="10" column="0" colspan="7">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>radTemplate</tabstop>
  <tabstop>cmbTemplates</tabstop>
  <tabstop>radCopy</tabstop>
  <tabstop>cmbBoxes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
